### PR TITLE
clarify adding external connections instructions in README, fix #837

### LINKED ISSUE #837: README install assumes use of WordPress <5.6 and use of application-passwords plugin
### Is your enhancement related to a problem? Please describe.

I found the [installation instructions in the README](https://github.com/10up/distributor#set-up-external-connections-using-application-passwords) fairly confusing since I am using WordPress 5.8.x (Application Passwords has been integrated into core since 5.6). Even with the application-passwords plugin installed, the directions did not quite match the UI that is presented by WordPress. 

Notably, the 'add external connection' screen by default just prompts you to enter the website's rest URL; there is no visible field for the application password. 
image

Once you click the authorize connection, you're brought to the login screen of the main site. I eventually figured out that this requires the main site's user name and user's password (NOT the application password) which is discouraged from a security perspective.  

I'll make a PR that hopefully more clearly explains how to use the application password 

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by skorasaurus**
### Description of the Change

Clarified the instructions for setting up External Connections. Existing instructions assumes use of the applications-password plugin whose functionality has been merged into WordPress core as of 5.6. 
The current `Add New External Connection` page is also different than what is described in the readme and instructions have been updated for that (more information in #837 

Closes #837 

### Possible Drawbacks

Maybe you want users to use the application-password plugin? Additional references to using the application-password are still in README and readme.

### Verification Process

What process did you follow to verify that your change has the desired effects?

 - Manually followed my new instructions on 5.8.3 websites. 

### Checklist:

- [ x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [x ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [ na] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Changelog Entry

> Changed - documentation updates.

### Credits

Props @skorasaurus, @jeffpaul.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,9 +73,10 @@ To help inform our roadmap, keep adopters apprised of major updates and changes
...
-1. On remotesite.com, ensure that [Application Passwords](https://wordpress.org/plugins/application-passwords/) is installed. (_Note: Using this plugin instead of a normal WordPress users password helps limit the use of your primary password and will allow you to revoke access to Distributor in the future if needed._) Then navigate to the user profile that will be used to create the External Connection on mainsite.com and then to the `Application Passwords` section of the user profile (not the `Account Management` section).  Add a label for the New Application Password Name (e.g., `mainsite.com`) and click `Add New`.  Now copy the password provided into mainsite.com's External Connections `Password` field.
-1. On mainsite.com, add the `External Connection URL` (e.g., http://remotesite.com/wp-json).  You should see a green circle and "_Connection established._".
+1. Enter a label for the connection (e.g., `remotesite.com`).
+1. Select `Username / Password` for the `Authentication Method`, open the `manually set up connection` link, and input a username from remotesite.com.
```
Comment (jeffpaul) [inline README.md]: This and the next couple steps can generally be skipped as the "Remote Connection Wizard" helps skip many of these steps.  Should essentially be (1) add label for the External Connection, (2) enter domain for the external site, (3) click Authorize Connection, (4) process acceptance from there to get to the final step of saving the new External Connection.

**Review by jeffpaul — CHANGES_REQUESTED** [review]
@skorasaurus thanks for getting this started, much appreciated!  Things should be even more simple than what you've updated to here, so if you're able to update your PR to the non-manual connection setup to use the Remove Connection Wizard steps that would be great... thanks again!

File: README.md (hunk 1 above)
Comment (skorasaurus) [reply README.md]: I think I had a misunderstanding of how Application Passwords were being used and did not think that an Application Password.

As I understand now, an application password is now automatically generated and saved when going through the "Remote Connection Wizard" process although the user first enters their user name and their password (not an application password) when making the authorization (if the user is not already logged into that other website). At the time of making the PR, I didn't know that (application password creation) was happening behind the scene. 

The message in the 'Edit External Connection' page - 
_"Important: We strongly recommend using the [Application Passwords](https://wordpress.org/plugins/application-passwords/) plugin on the site you are connecting to in order to create a unique password for this connection. This helps limit the use of your primary password and will allow you to revoke access in the future if needed."_  - 

led me to believe that my credentials were being stored instead (of a unique application password) ; as indicated in the screenshot below

Selection_148

I've updated the PR using the "Remote connection Wizard" directions.

Comment (jeffpaul) [reply README.md]: @skorasaurus good callout on the text on the Edit External Connection page, we could/should probably remove that entire line at this point as that adds unnecessary plugins to someone's site as WordPress core now handles all that functionality.  If you'd like, I'd say go ahead and strip that out as part of this PR as well if you'd like?

File: README.md [hunk 2]
```diff
@@ -69,14 +69,15 @@ You can upload and install the archived (zip) plugin via the WordPress dashboard
...
-1. On mainsite.com, add the `External Connection URL` (e.g., http://remotesite.com/wp-json).  You should see a green circle and "_Connection established._".
-1. Ensure the roles selected in `Roles Allowed to Push` are the ones you want to support, then press the `Create Connection` button.  You should now be able to push from mainsite.com to remotesite.com.  If you want to pull from remotesite.com to mainsite.com, simply repeat these instructions swapping mainsite.com and remotesite.com.
+1. On remotesite.com, navigate to `Distributor` > `External Connections` and click `Add New`.
```
Comment (jeffpaul) [inline README.md]: As the primary use case with Distributor is a main (or HQ/headquarters or origin) site that connects out to other sites to Push content, was there a reasoning you changed these instructions to connect from a remote site back to the main site?  If not, swapping those references would be more in line with our expected primary use case with Distributor and be good to merge in... thanks!

Comment (skorasaurus) [reply README.md]: pulling in content from remote to main had been our primary use case; I now updated it to reflect from main to remote as you wished.

**Review by jeffpaul — APPROVED** [review]
Thanks @skorasaurus, much appreciated!

**Pull request merged**